pkg/target/http.go

```diff
@@ -268,13 +313,12 @@ func (ht *HTTPTarget) Write(messages []*models.Message) (*models.TargetWriteResu
 			var badMsgs []*models.Message
 			var err error
 
-			// just for now to spike
-			templaterConfigured := false
-			if templaterConfigured {
-				reqBody, goodMsgs, badMsgs, err = ht.requestTemplater(templ, group)
+			if ht.requestTemplate != nil {
+				reqBody, goodMsgs, badMsgs, err = renderBatchUsingTemplate(ht.requestTemplate, group)
 			} else {
 				reqBody, goodMsgs, badMsgs, err = ht.provideRequestBody(group)
```

Comment:
nit: one is a member of ht and the other is just a free-floating function. I don't think there's any argument for one over the other but something about the mismatch bugs me